docs/providers/aws/guide/serverless.yml.md

```diff
@@ -136,8 +136,10 @@ provider:
     lambda: true # Optional, can be true (true equals 'Active'), 'Active' or 'PassThrough'
   logs:
     restApi: # Optional configuration which specifies if API Gateway logs are used. This can either be set to true to use defaults, or configured via subproperties.
-      format: 'requestId: $context.requestId' # Optional configuration which specifies the log format to use.
-      level: INFO # Optional configuration which specifies the log level to use. May be either INFO or ERROR.
+      enableAccessLogging: true # Optional configuration which enables or disables access logging. Defaults to true.
+      format: 'requestId: $context.requestId' # Optional configuration which specifies the log format to use for access logging.
+      level: INFO # Optional configuration which specifies the log level to use for execution logging. May be set to either INFO or ERROR, or set to OFF to disable.
+      dateTrace: true # Optional configuration which specifies whether or not to log full requests/responses for execution logging. Defaults to true.
```

Comment:
Sure, that's more descriptive, I'll change it.

Reply:
As we use `executionLogging` instead of `dataLogging`. Maybe it'll be better to call it `fullExecutionData`.. it'll also give hint, that it affects only _execution logging_

Reply:
Good call, I've updated it.

Reply:
When PR will be ready for re-review, just press that _refresh_ button (re-request review) in reviewers section.
Thanks!

Reply:
It's ready for review; however I don't see the refresh button next to your name (as in https://help.github.com/en/articles/requesting-a-pull-request-review) in the reviewers section. I only see the red +/- icon there.

Reply:
Ok, it's probably accessible for members only. Sorry for confusion, I thought it's always avaialable to PR authors

Reply:
Haha, no problem. It's most likely that way to prevent abuse by spamming review requests. It is a bit inconvenient though, would be nice if it was available to everyone.